### DISCUSSION TITLE
Running tests on local computer

### DISCUSSION THREAD

**Discussion by raoulcollenteur** [pastas/pastas#369] [General]
I am trying to run the test suit on my local computer, but running into some problems. I think this is due to pytest.ini. 

@dbrakenhoff do you know how to run the tests locally now?

Cheers,
Raoul

**Comment by dbrakenhoff** [comment]
Does it have to do with the coverage statements in `pytest.ini`? I.e. you need the packages `pytest-cov` and `codecov` which maybe are note installed by default?

But simplest fix is probably to rename the pytest.ini file to something else and it will be ignored.
